corehq/apps/es/queries.py

```diff
@@ -111,6 +113,18 @@ def match(search_string, field, fuzziness="AUTO", operator=None):
     }
 
 
+def fuzzy(search_string, field, fuzziness="AUTO"):
+    return {
+        "fuzzy": {
```

Comment:
Did this end up behaving differently than a `match` query with `fuzziness` set?  I don't recall.

Reply:
I think they behave the same except that the `match` query doesn't allow modifying `max_expansions`.